runtime/lua/vim/lsp/completion.lua

```diff
@@ -265,7 +270,7 @@ function M._lsp_to_complete_items(result, prefix, client_id)
       end
       local completion_item = {
         word = word,
-        abbr = item.label,
+        abbr = vim.trim(item.label),
```

Comment:
Not sure about this trim. If there's leading whitespace I feel like either the label is incorrect or the space is intentional.

Reply:
https://github.com/llvm/llvm-project/blob/7ba6768df8181bc270763333969d4a1d6cc2e160/mlir/lib/Tools/mlir-pdll-lsp-server/PDLLServer.cpp#L709

there is no trim so we can assume that the server did this on purpose?

Reply:
clangd prefixes some items with a •, so I assume the space is to keep all items aligned:

```
 a                                                                Variable   std::unique_ptr<A>
 A                                                                Struct
•ADJ_ESTERROR                                                     Text
•ADJ_FREQUENCY                                                    Text
```